app/pdl.yaml

```diff
@@ -15,16 +15,28 @@ tags:
   description: "Farm Bill summary data."
 - name: "All Programs"
   description: "All programs data."
-- name: "Title I: Commodities"
-  description: "Farm Bill Title I: Commodities data."
-- name: "Title II: Conservation - CSP"
-  description: "Farm Bill Title II: Conservation - Conservation Stewardship Program (CSP) data."
-- name: "Title II: Conservation - EQIP"
-  description: "Farm Bill Title II: Conservation - Environmental Quality Incentives Program (EQIP) data."
-- name: "SNAP State Distribution Data"
-  description: "Supplemental Nutrition Assistant Program (SNAP) state distribution data"
 - name: "States"
   description: "Information related to States."
+- name: "Title I: Commodities - Subtitle A"
+  description: "Title I Subtitle A commodities data."
+- name: "Title I: Commodities - Subtitle D, Dairy Margin Coverage"
+  description: "Title I Subtitle D Dairy Margin Coverage data."
+- name: "Title I: Commodities - Subtitle E, Supplemental Agricultural Disaster Assistance"
+  description: "Title I Subtitle E Supplemental Agricultural Disaster Assistance data."
+- name: "Title II: Conservation - EQIP (Environmental Quality Incentives Program)"
+  description: "Title II EQIP conservation data."
+- name: "Title II: Conservation - CSP (Conservation Stewardship Program)"
+  description: "Title II CSP conservation data."
+- name: "Title II: Conservation - CRP (Conservation Reserve Program)"
+  description: "Title II CRP conservation data."
+- name: "Title II: Conservation - ACEP (Agricultural Conservation Easement Program)"
+  description: "Title II ACEP conservation data."
+- name: "Title II: Conservation - RCPP (Regional Conservation Partnership Program)"
+  description: "Title II RCPP conservation data."
+- name: "Title XI: Crop insurance"
+  description: "Title XI Crop Insurance data."
+- name: "Title IV: SNAP (Supplemental Nutrition Assistant Program)"
```

Comment:
It may be better to put Title IV name and description above Title XI for arranging the endpoints in the increasing order titles.

Reply:
You're right. Modified